### ISSUE TITLE
Solr package out-of-the-box not working 

### ISSUE THREAD

**Issue by MustafaKarabulut** [iorixxx/lucene-solr-analysis-turkish#11]
Hi,

Thank you for your work. 

I downloaded the ready to use Solr package you provided on the front page (https://www.dropbox.com/s/yygdvwoe4cc7d46/solr-7.3.0.tgz). I expected it to run out-of-the-box.

When I try to start Solr, it hangs with the below messages:
![resim](https://user-images.githubusercontent.com/24882371/83356719-828a7c80-a370-11ea-8b98-e149bbc9c1b8.png)

In the meantime, the Solr web interface gives this error:
![resim](https://user-images.githubusercontent.com/24882371/83356732-a221a500-a370-11ea-8c8a-64d429579fc9.png)

Java 1.8 is installed on my machine.

Is there anything else I should do?

Thanks in advance

**Comment by iorixxx** [comment]
I couldn't reproduce this. I downloaded solr-730.tar.gz, untar it. Then run the command : **bin/solr start -f**
It works for me. java version "1.8.0_60" 
Can you look into your logs and post the error exception messages?

**Comment by MustafaKarabulut** [comment]
Thank you for your reply.

The log file at solr-7.3.0/server/logs/solr.log is attached: 
[solr.log](https://github.com/iorixxx/lucene-solr-analysis-turkish/files/4734967/solr.log)

The startup process stalls without any exception/error message after the last line in the above log file is printed on the screen.

As a side note, the timestamp in the log file is wrong (3 hours behind the actual time). I am not sure whether it is important or not though.

**Comment by iorixxx** [comment]
As you said, log file includes WARN and INFO only. I think you are using Windows OS, that might be the difference. But I don't have a Windows box/machine; thus, I cannot test it on Windows. Just a guess, it might a file separator thing. Can't you just use a newer version of solr?

**Comment by MustafaKarabulut** [comment]
Thank you @iorixxx 

It worked in Ubuntu without any problems. As you told, it looks like an operating system compatibility problem that won't allow it work under Windows.

**Comment by MustafaKarabulut** [comment]
UPDATE:
Even though the Solr 7.3 package did seem to start on Ubuntu, I lived some problems with it while adding documents (e.g., it gives some specific errors about its not being able to index). So, I have given up on Solr 7.3.

Afterwards, I got a new version of Solr (8.5) on my Windows machine and copied jars/dic files into necessary places. As a result, it works (starts and indexes documents as expected)!
